SSDAlignment/SSDAlign.h

```diff
@@ -9,11 +9,13 @@
 #include <vector>
 #include <stdint.h>
 #include <iostream>
+#include "TVector3.h"
```

Comment:
is this needed? i don't see where it would be called

Reply:
They are unnecessary, removed.